server/tokenapi/Main.go

```diff
@@ -0,0 +1,50 @@
+package tokenapi
+
+import (
+	"github.com/aerogo/aero"
+	"github.com/aerogo/log"
+	"github.com/akyoto/uuid"
+	"github.com/animenotifier/notify.moe/arn"
+	"github.com/tidwall/gjson"
```

Comment:
Might want to double-check that another JSON lib is required here.
Clarify in a GitHub comment why we absolutely need another package, or use the existing ones (preferred).

Reply:
Oh yeah, that was mostly because I'm too used to Gjson since I dislike the standard json implementation. I'll probably switch to the standard one sooner or later.